databend_py/client.py

```diff
@@ -271,6 +271,21 @@ def upload(self, file_name, table_name, file_type=None):
         file_data = self.get_file_data(file_name)
         self.sync_csv_file_into_table(file_name, file_data, table_name, file_type)
 
+    def upload_to_stage(self, file_name):
+        """
+        upload the file to user stage
+        :param file_name:
+        :return:
+        """
+        file_data = self.get_csv_data(file_name)
```

Comment:
the file format might differs from csv, I guess we do not need read out the file's content here, just pass the file descriptor to `requests` is also works.

```
requests.put(presigned_url, header=headers, data=f)
```

Reply:
Use `requests.put(presigned_url, header=headers, files=f)` can upload the file but the uploaded is like :
![fefKqkN7pO](https://github.com/databendcloud/databend-py/assets/7600925/3e983dec-66c8-49df-a022-38702e62404c)


So when copy into file like this will get an error:

![a00a35b2-a635-4e68-9119-4caf6ca6380b](https://github.com/databendcloud/databend-py/assets/7600925/2d7abbac-2fb6-46b6-bc16-019981434b8e)